### PR TITLE
New migration: fenrir_id column added.

### PR DESCRIPTION
**Pull request by lrocheWB**
# Introduction
We need to do a link between NMM customers and users Fenrir. This migration add "fenrir_id" column in table customers and create users in fenrir-api with identifier of customer.

## Dependencies
Depends of:
- [x] https://github.com/CanalTP/MigrationBundle/pull/1
- [ ] https://github.com/CanalTP/NMM/pull/182

## Testing
```
php app/console claroline:migration:upgrade CanalTPNmmPortalBundle
php app/console claroline:migration:downgrade CanalTPNmmPortalBundle

### CODE REVIEW

File: Migrations/pdo_pgsql/Version003.php [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+        return $statement->fetchAll();
+    }
+
+    public function down(Schema $schema)
+    {
+        $this->addSql('ALTER TABLE public.tr_customer_cus DROP COLUMN fenrir_id;');
+
+        foreach ($this->fenrirClient->getUsers() as $user) {
+            $this->fenrirClient->deleteUser($user->id);
```
Comment (Netmisa) [inline Migrations/pdo_pgsql/Version003.php]: You remove all users (including from all origin).
You need to add filter by self::NMM_ORIGIN

**Comment by bbonnesoeur** [comment]
@Netmisa Just a though about fenrir, is it still possible to have an NMM instance working whitout a fenrir instance ?

**Comment by Netmisa** [comment]
In fact no, we need to install fenrir with NMM (exactly like tyr).
This PR https://github.com/CanalTP/NMM/pull/182 will be create the fenrir dependence.

**Comment by bbonnesoeur** [comment]
As we discussed, NMM and underlying component were concepted to be TYR / Navitia and other business component CTP powered independent. For example NMM / Sam was designed in order to be able to be a backend for a completly different businness app. 
Moreover  there is a param to choose if you have a Tyr Installed or not.  If not, As far as I remember, the customer creation form ask for token instead of calling Tyr API.

**Pull request merged**